plugins/fused-ops-and-kernels/configs/fast_kernels.yaml

```diff
@@ -22,4 +22,7 @@ training:
     fast_rms_layernorm: True
 
     # fast RoPE embedding triton kernels
-    fast_rope_embeddings: True
\ No newline at end of file
+    fast_rope_embeddings: True
+
+    # fused linear cross entropy loss
```

Comment:
maybe but some more comments on how setting this will true will require fast_loss setting to be overrrden, or invalidated. or whatever we deciode